docs/security/authentication/index.asciidoc

```diff
@@ -100,7 +100,7 @@ xpack.security.authc.saml.realm: realm-name
 +
 [source,yaml]
 --------------------------------------------------------------------------------
-server.xsrf.whitelist: [/api/security/v1/saml]
+server.xsrf.whitelist: [/api/security/authc/saml/callback]
```

Comment:
__note:__ previous comment was vanished with new commit. But essential I wasn't sure whether we need `authc` part here. The more I think about it the more it seems unnecessary....